src/Serilog/Events/LogEventProperty.cs

```diff
@@ -39,6 +39,113 @@ public LogEventProperty(string name, LogEventPropertyValue value)
             Value = value;
         }
 
+#if NET6_0_OR_GREATER
+        /// <inheritdoc />
+        public LogEventProperty(string name, TimeOnly value) : this(name, new ScalarValue(value))
```

Comment:
Perhaps naming the `value` argument in these overloads `scalarValue` would make the difference between them and the original constructor clearer?